peps/pep-9999.rst

```diff
@@ -20,38 +20,41 @@ This PEP specifies a name mapping mechanism that allows packaging tools to map
 canonical PyPI and external dependency identifiers to their counterparts
 in other package repositories.
 
+.. JRG: Are we mapping PyPI identifiers here? I don't think so, right? I think the
+   abstract should just say "... to map external dependency identifiers (as introduced
+   in PEP 725) to..."
```

Comment:
After going through the Examples section and adding [this note](https://github.com/rgommers/peps/pull/39/commits/9296ce13422532df1d2e412ac07a836d8d7fc034), I'm not so sure anymore. Can you clarify here @rgommers? If we want to do this, we need to:

- Add a bunch of `dep:pypi/*` URLs to the central registry (build backends, and other packages often found in `[build-system]`)
- Teach `pyproject-external` to read `[build-system]` dependencies, convert them to `dep:pypi/*` and enable their display in the CLI. May need to rethink the current API centered around the `[external]` table, too.

Reply:
Yes, let me point to this example in the `pep-name-mapping`:

https://github.com/rgommers/peps/blob/1dabd6b9c09b8cb15e01fba3df2fbfed10806e47/peps/pep-9999.rst?plain=1#L672-L675

The Python deps in build-system are being mapped to `apt` package names too.

Reply:
> But those `ids` are not "canonical" (no pun intended) in the sense that they are not in the central registry.

Agreed. Best not to mention it at all.

Reply:
Got it. I'll remove all these references then.